sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClient.java

```diff
@@ -40,16 +39,13 @@
 public class EventHubClient implements Closeable {
```

Comment:
Make this package-private.

Reply:
That's in another PR. it breaks a lot of things. After this is merged, I'll create another that makes it package private and updates it.